packages/ui-kit/src/hooks/useTopValues/useTopValues.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import type { Meta, StoryObj } from '@storybook/react'
+import { Chart, registerables } from 'chart.js'
+
+import axiosInstance from '../../../../../app/storybook/src/axios'
+import { storybookCodeTemplate, RelativeTimeRange, useStorybookAccessToken } from '../../helpers'
+
+import { useTopValues } from './useTopValues'
+import { TopValuesQueryProps } from '../../components/TopValues/TopValues.types'
+
+Chart.register(...registerables)
```

Comment:
By using `registerables`, we do indeed register everything from Chart.js, which may help avoid issues related to unregistered parts. However, this approach also negates the benefits of tree-shaking. Can we register only the required things here?